# SpaT.msg
uint8 id
uint8 current_state
uint32 current_duration
uint8 next_state
